### ISSUE TITLE
Camera device bandwith issue

### ISSUE THREAD

**Issue by tikismoke** [domogik/domoweb#90]
Ok all this bandwith comes from my Zoneminder container.
making also the cpu grow for this container.
Camera widget are not displayed on domoweb but it seems it make no difference and it grab the camera image even not to display them
All this mixed make:
My wifi network fall down
My cpu (pi and serveur) used grow for no reason as they are not displayed
zoneminder container eth0 traffic (down is camera catch image) up is client video: https://snag.gy/7c2hv1.jpg
Pi2 bandwith https://snag.gy/eZxCqm.jpg every time i display domoweb on chromium it make bandwith >1M
As you can see zoneminder could send @20M but the pi is limit to 10M so sometimes camera are no more available on domoweb, and in all case as i do not display them at same all time it should not do that.

**Comment by overload08** [comment]
This issue is related to pi hardware.
I've done some widget as your demand for refresh and fullscreen. Is it ok ?

**Comment by tikismoke** [comment]
Not only a pi, I get same problems on all device until bandwidth is full (androids too).

The widget should unregesister the video flow when it disappears from the current view.

**Comment by overload08** [comment]
Ok, i see.
if you have 4 sections, with 4 differents videos on each, you have 4x bandwidth ?

**Comment by tikismoke** [comment]
Yes and it will increase when opening a previous section to grab again a new video flow. I'll explain you on irc.

Le 1 février 2017 20:37:33 GMT+01:00, OverLoad <notifications@github.com> a écrit :
>Ok, i see.
>if you have 4 sections, with 4 differents videos on each, you have 4x
>bandwidth ?
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/domogik/domoweb/issues/90#issuecomment-276758419

-- 
Envoyé de mon appareil Android avec K-9 Mail. Veuillez excuser ma brièveté.

**Comment by overload08** [comment]
ok, i'm waiting you
